[all] UT Test failed with conversion from int to string yields a string of one rune

<!--
Please add the affected binary name in the title unless multiple binaries are affected, e.g.
[cinder-csi-plugin] Cannot delete PV
For openstack-cloud-controller-manager, you can use [occm] for short.

All the currently maintained binaries are:
* openstack-cloud-controller-manager (occm)
* cinder-csi-plugin
* manila-csi-plugin
* k8s-keystone-auth
* client-keystone-auth
* octavia-ingress-controller
* magnum-auto-healer
* barbican-kms-plugin
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature

**What happened**:

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/cloud-provider-openstack/1115/pull-cloud-provider-openstack-check/1288650530599473153/build-log.txt
```
W0730 01:41:36.108] pkg/ingress/controller/openstack/neutron.go:216:23: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
W0730 01:43:05.362] make: *** [Makefile:182: vet] Error 2
W0730 01:43:05.362] Traceback (most recent call last):
W0730 01:43:05.363]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 50, in <module>
W0730 01:43:05.363]     main(ARGS.env, ARGS.cmd + ARGS.args)
W0730 01:43:05.364]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 41, in main
W0730 01:43:05.364]     check(*cmd)
W0730 01:43:05.364]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 30, in check
W0730 01:43:05.365]     subprocess.check_call(cmd)
W0730 01:43:05.365]   File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
W0730 01:43:05.365]     raise CalledProcessError(retcode, cmd)
W0730 01:43:05.367] subprocess.CalledProcessError: Command '('make', 'check')' returned non-zero exit status 2
```
**What you expected to happen**:


**How to reproduce it**:


**Anything else we need to know?**:


**Environment**:
- openstack-cloud-controller-manager(or other related binary) version:
- OpenStack version:
- Others:

--- a/pkg/ingress/controller/openstack/neutron.go
+++ b/pkg/ingress/controller/openstack/neutron.go
@@ -213,7 +213,7 @@ func (os *OpenStack) EnsureSecurityGroupRules(sgID string, sourceIP string, dstP
 	// Because the security group is supposed to be managed by octavia-ingress-controller, we assume the `port_range_min`
 	// equals to `port_range_max`.
 	for _, rule := range allRules {
-		if !dstPortsSet.Has(string(rule.PortRangeMin)) {
+		if !dstPortsSet.Has(strconv.Itoa(rule.PortRangeMin)) {
 			// Delete the rule
 			if err := rules.Delete(os.neutron, rule.ID).ExtractErr(); err != nil {
 				return err

